README.md

```diff
@@ -126,11 +126,17 @@ the bus:
   `com.redhat.Yggdrasil1.Worker1` and includes its directive as the final segment
   in reverse-domain-name notation (i.e. `com.redhat.Yggdrasil1.Worker1.echo`).
 
-A worker can transmit data back to a destination by calling the
-`com.redhat.Yggdrasil1.Dispatcher1.Transmit` method.
+A worker can transmit data back to a destination by calling either the
+`com.redhat.Yggdrasil1.Dispatcher1.Transmit` or the
+`com.redhat.Yggdrasil1.Dispatcher1.Request` method. "Transmit" sends data back
+across the configured network transport (typically MQTT), while "Request"
+creates an HTTP request and sends it to the requested URL. Which method to use
+is left to worker discretion, though it is recommended that messages larger than
+a few megabytes be sent using HTTP.
 
 Package `worker` implements the above requirements implicitly, enabling workers
 to be written without needing to worry about much of the D-Bus requirements
 outlined above.
 
-See `worker/echo` for a reference implementation of a worker program.
+See `worker/echo` for a reference implementation of a worker program that uses
+"Transmit". See `worker/http` for a demonstrative worker that uses "Request".
```

Comment:
I would add somewhere example how to send MQTT message to new http worker. Example would be:

```
echo '{"method": "GET", "url": "http://httpbin.org/get", "headers": {"User-Agent": "yggdrasil-http-worker"}}' | \ 
    go run ./cmd/yggctl generate data-message --directive http - | \
    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/data/in
```

Reply:
I feel like this kind of detail belongs more in the CONTRIBUTING document,
rather than the README. The README is intended for end users more than
contributors. CONTRIBUTING contains existing instructions on how to interact
with workers; maybe I can add a section in that document on how to interact with
the `http` worker.

Reply:
I added a section to the `http` worker README.